src/lib.rs

```diff
@@ -1280,7 +1280,7 @@ impl ExpressionStatus {
     }
 }
 
-mod errors {
+pub mod errors {
```

Comment:
Yep. You're right. I feel stupid now. 😞
Sorry for all the noise. Releasing the package is enough. 👍